config/clusters/hhmi/common.values.yaml

```diff
@@ -0,0 +1,66 @@
+basehub:
+  nfs:
+    enabled: true
+    dirsizeReporter:
+      enabled: true
+    pv:
+      mountOptions:
+        - soft
+        - noatime
+      serverIP: 10.55.112.74
+      baseShareName: /homes/
+  jupyterhub:
+    custom:
+      2i2c:
+        add_staff_user_ids_to_admin_users: true
+        add_staff_user_ids_of_type: "github"
+      homepage:
+        templateVars:
+          org:
+            url: https://www.hhmi.org/
+            logo_url: https://www.hhmi.org/sites/default/files/Logos/JPG/HHMI-vertical-signature-color.jpg
+          designed_by:
+            name: 2i2c
+            url: https://2i2c.org
+          operated_by:
+            name: 2i2c
+            url: https://2i2c.org
+          funded_by:
+            name: TODO
+            url: https://www.hhmi.org/
+    singleuser:
+      # User image: https://quay.io/repository/arokem/nh2023?tab=tags
+      image:
+        name: pangeo/pangeo-notebook
+        tag: "2023.09.11"
+      nodeSelector:
+        node.kubernetes.io/instance-type: n2-highmem-16
+      cpu:
+        guarantee: 0.5
+        # We're on n1-highmem-16 machines
+        limit: 14
```

Comment:
This probably needs a G after

Reply:
@yuvipanda, I think it's ok because this is the cpu limit. I just double-checked and the memory guarantee and limit have the G.